packages/shared/src/lib/styling.ts

```diff
@@ -0,0 +1,16 @@
+import classNames from 'classnames';
+
+const fadedBackground =
+  'relative before:absolute before:inset-0 before:w-full before:h-full before:opacity-24';
+
+export const getFadedBackground = (className: string): string => {
+  const isBefore = className.includes('before:bg-theme-');
+
+  if (!isBefore) {
+    throw new Error(
+      'You must need apply the background with a `before:` class',
```

Comment:
```
'You need to apply the background with a \'before:\' class'
```

Reply:
The lint gave an error. Let me make the parent a double quotation as the lint suggested. Thanks for mentioning!